### PR TITLE
feature(amp-live-list): add data-tombstone feature

### PR DESCRIPTION
**Pull request by erwinmombay**
follow up on limiting count of items here https://github.com/ampproject/amphtml/issues/3330 (issue to respect the max-items-per-page value)

### CODE REVIEW

**Comment by erwinmombay** [comment]
@cramforce PTAL

File: extensions/amp-live-list/0.1/amp-live-list.js [hunk 1]
```diff
@@ -269,14 +286,35 @@ export class AmpLiveList extends AMP.BaseElement {
...
+   * @private
+   */
+  tombstone_(parent, orphans) {
+    orphans.forEach(orphan => {
+      const orphanId = orphan.getAttribute('id');
+      const liveElement = parent.querySelector(`#${orphanId}`);
+      if (!liveElement) {
+        return;
+      }
+      liveElement.setAttribute('data-tombstone', '');
```
Comment (cramforce) [inline extensions/amp-live-list/0.1/amp-live-list.js]: Add comment: Triggers `display:none` style.

File: extensions/amp-live-list/0.1/live-list-manager.js [hunk 2]
```diff
@@ -62,6 +63,16 @@ export class LiveListManager {
       // Switch out the poller interval if we can find a lower one and
       // then make sure to stop polling if viewer is not visible.
       this.interval_ = Math.min.apply(Math, this.intervals_);
+
+      // For testing purposes only, we speed up the interval of the update.
+      // This should NEVER be allowed in production.
+      const isUpdateExample = this.win.location.pathname == '/examples.build/' +
```
Comment (cramforce) [inline extensions/amp-live-list/0.1/live-list-manager.js]: I'd inline this expression as it is only used once.

**Comment by cramforce** [comment]
LGTM

Very nice!

File: extensions/amp-live-list/0.1/amp-live-list.js (hunk 1 above)
Comment (erwinmombay) [inline extensions/amp-live-list/0.1/amp-live-list.js]: done.

**Comment by erwinmombay** [comment]
@cramforce qq, when we do the limit number of items to `max-items-per-page` code do we mark and remove the subtree like the  `data-tombstone` mechanism, or should we just outright remove those element items from the live DOM.

**Comment by cramforce** [comment]
@erwinmombay We should use the tombstone mechanism only per semantically deleted rows. If we throw away over max-item we should just keep a map of ids for those, but no remnants in the DOM.

File: extensions/amp-live-list/0.1/live-list-manager.js (hunk 2 above)
Comment (erwinmombay) [inline extensions/amp-live-list/0.1/live-list-manager.js]: done.

**Pull request merged**